db/migrate/20160119125950_add_created_on_for_container_entities.rb

```diff
@@ -0,0 +1,16 @@
+class AddCreatedOnForContainerEntities < ActiveRecord::Migration
+  CONTAINER_TABLES = [:container_nodes, :container_projects, :container_services, :container_routes, :container_groups,
+                      :container_replicators, :container_quotas, :container_builds, :container_build_pods,
+                      :container_limits, :container_volumes]
+
+  def change
+    CONTAINER_TABLES.each do |t|
+      add_column t, :created_on, :datetime
+      rename_column t, :creation_timestamp, :created_on_ems
```

Comment:
`vms` has an `ems_created_on` column; if we're favouring consistency over correctness, we should probably be consistently consistent.